lib/src/localization.dart

```diff
@@ -13,6 +13,15 @@ class Localization {
   String path;
   bool useOnlyLangCode;
   final RegExp _replaceArgRegex = RegExp(r'{}');
+  final RegExp _linkKeyMatcher =
+      RegExp(r'(?:@(?:\.[a-z]+)?:(?:[\w\-_|.]+|\([\w\-_|.]+\)))');
+  final RegExp _linkKeyPrefixMatcher = RegExp(r'^@(?:\.([a-z]+))?:');
+  final RegExp _bracketsMatcher = RegExp(r'[()]');
```

Comment:
brakets work like in vue? `@:(message.foo.bar.baz)`?

Reply:
need test

Reply:
Yes, exactly like that. And I've already added tests in my PR.